### PR TITLE
💄 Improvements to ticketer connect views

### PR DESCRIPTION
**Pull request by rowanseymour**
Captura de Pantalla 2020-06-04 a la(s) 14 58 43

Captura de Pantalla 2020-06-04 a la(s) 14 58 59

Captura de Pantalla 2020-06-04 a la(s) 14 59 13

### CODE REVIEW

File: temba/tickets/types/mailgun/type.py [hunk 1]
```diff
@@ -18,15 +18,12 @@ class MailgunType(TicketerType):
...
 
     connect_view = ConnectView
     connect_blurb = _(
-        """Connecting an email address will forward all new tickets and their responses to that email address. You will be able to respond to them directly and your replies will be sent to the contact."""
-    )
-
-    form_blurb = _(
-        """New tickets and replies will be sent to the email address that you configure below. You will need to verify it by entering the token sent to you."""
+        "Connecting an email address will forward all new tickets and their responses to that email address. "
+        "You will be able to respond to them directly and your replies will be sent to the contact."
```
Comment (rowanseymour) [inline temba/tickets/types/mailgun/type.py:26]: this is way to make a `_()` string which 1) is trimmed and 2) doesn't exceed the line length

File: temba/tickets/types/zendesk/views.py [hunk 2]
```diff
@@ -106,9 +111,6 @@ def handle_code_granted(self, request, *args, **kwargs):
             config=config,
         )
 
-        # TODO: set up trigger on Zendesk side to callback to us on ticket closures
-        # See: https://developer.zendesk.com/rest_api/docs/support/triggers
```
Comment (rowanseymour) [inline temba/tickets/types/zendesk/views.py:110]: mailroom does this

**Comment by nicpottier** [comment]
How do you feel about changing the page title `Connect Email` on step 2 to something like `Verify Email`? Those pages look pretty similar and when I went through it it took me a bit to even realize I had been taken somewhere different. (me typoing my email address notwithstanding)

**Comment by rowanseymour** [comment]
@nicpottier I had the same thought! Will update

**Comment by rowanseymour** [comment]
also to be clear the icon choice here is the usual me picking the first thing on icomoon.. just need something emaily that isn't the existing email icon we use for other email stuff

**Pull request merged**